xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,888 @@
+   </note>
+   <variablelist>
+    <varlistentry>
+     <term>External</term>
+     <listitem>
+      <para>
+       These machines will host the <literal>&rmtool;</literal> for RPM packages
+       and the <literal>Docker Pull-Trough Cache</literal> for container
+       images.
+      </para>
+      <itemizedlist>
+       <listitem>
+        <para>
+         <literal>2</literal> Host machines for the mirror servers.
+        </para>
+        <itemizedlist>
+         <listitem>
+          <para>
+           SLES 15
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           2 (v)CPU
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           8 GB RAM
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           500 GB Storage
+          </para>
+         </listitem>
+        </itemizedlist>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Internal</term>
+     <listitem>
+      <para>
+       These machines will host the <literal>&rmtool;</literal> for RPM packages,
+       and <literal>Docker Container Registry</literal> for container images
+       as well as a webserver hosting the <literal>Helm Chart Repository</literal>
+       files.
+      </para>
+      <itemizedlist>
+       <listitem>
+        <para>
+         <literal>2</literal> Host machines for the mirror servers.
+        </para>
+        <itemizedlist>
+         <listitem>
+          <para>
+           SLES 15
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           2 (v)CPU
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           8 GB RAM
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           500 GB Storage
+          </para>
+         </listitem>
+        </itemizedlist>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+   <important>
+    <title>Adjust Number Of Mirror Servers</title>
+    <para>
+     This scenario description does not contain any fallback contingencies for
+     the mirror servers. Add additional mirror servers behind a loadbalancer if
+     you require additional reliability/availability.
+    </para>
+   </important>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.network">
+   <title>Networking</title>
+   <para>
+    All members of the &productname; cluster must be able to communicate with
+    the internal mirror server(s) within the airgapped network.
+   </para>
+   <para>
+    Ports:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      80 HTTP - RMT Server and Helm Chart Repository Mirror
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      443 HTTPS - RMT Server and Helm Chart Repository Mirror
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      5000 HTTP - Docker Registry
```

Comment:
Question: should we recommend to use a TLS secured docker registry?

Reply:
If we know how to do that, of course. The secure option is always preferrable to customers over the unsecured one.

Reply:
We can use the info from Vicentes tests: https://github.com/chentex/registry-testing